projects/cashmere-examples/src/lib/input-zipcode/input-zipcode-example.component.html

```diff
@@ -0,0 +1,30 @@
+<form>
+    <div class="form-container">
+        <hc-form-field>
+            <hc-label>Zip Code:</hc-label>
+            <input hcInput hcZipCodeMask
+                   [formControl]="formDemoZip"
+                   [preValue]="formDemoZip.value"
+                   [zipControl]="formDemoZip">
+            <hc-error>A valid zip code is required</hc-error>
+        </hc-form-field>
+
+        <hc-form-field>
+            <hc-label>Zip Code (with initial value):</hc-label>
+            <input hcInput hcZipCodeMask
+                   [formControl]="formDemoZipInitVal"
```

Comment:
Is there a way to make it so that it doesn't require the formControl and the zipControl (eliminate the zipControl) if possible since they should always be the same. I think I did this on the currency control (or one of those).

Reply:
This has been fixed with the latest commit.